all.json

```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
```

Comment:
Blocked already?

Reply:
removed the subdomain entry, as the main domain was already blocked, my bad (i opened it on a browser without the polka extension)